src/plugins/github/README.md

```diff
@@ -0,0 +1,175 @@
+# GitHub Plugin
+
+This is the SourceCred GitHub plugin. It creates a cred graph based on data
+from [GitHub].
+
+[GitHub]: https://github.com/
+
+## Nodes
+
+The GitHub plugin creates the following kinds of nodes:
+
+### Repo
+
+A GitHub repository, e.g. [sourcecred/sourcecred]. It will be directly
+connected to all of the pulls and issues in the repository. Setting a weight on
+the repository will have no effect (i.e. they do not mint cred). This will
+change when we switch to [CredRank].
```

Comment:
I think it's important to phrase this with a focus on the timestamp:
> The node **does not have a timestamp**, and therefore does not mint cred regardless of it's node weight. This can change with CredRank.

Because a `timestampMs: null` is a relevant detail of the output of this plugin. Whereas the other part is how nulls are interpreted.

Reply:
Done.